seqr/management/commands/run_postgres_database_backup.py

```diff
@@ -32,10 +40,12 @@ def run(cmd):
             backup_filename = "{db_name}_{deployment_type}_backup_{timestamp}.txt.gz".format(
                 db_name=db_name, deployment_type=options['deployment_type'], timestamp=timestamp)
 
-            run("/usr/bin/pg_dump -U postgres --host {postgres_host} {db_name} | gzip -c - > {backup_dir}/{backup_filename}".format(
+            run("pg_dump -U postgres --host {postgres_host} {db_name} | gzip -c - > {backup_dir}/{backup_filename}".format(
```

Comment:
I'm pretty sure taking this out causes this command to fail in our cron job

Reply:
good point. We should double check that /usr/bin and /usr/local/bin are on PATH in cron.

Reply:
I'm pretty sure they weren't which is why backups were failing that time we lost all the data :(